docs/actions/append_report.md

```diff
@@ -13,15 +15,16 @@ Priority helps with sorting phases and table content by becoming part of the fil
 
 **phase:** *required*, *no default:*  
 The phase, for which we generate the report. Currently, there are three phases:
+
 - install
 - test
 - report
 
 **job:** *not required*, *default:* -  
-The name of the job, e.g, "phpunit", "codeception" etc.
+The name of the job, e.g. "phpunit", "codeception" etc.
```

Comment:
My gut told me, that Grammarly would like a few more commas here and it was right...
`The name of the job, e.g., "phpunit", "codeception", etc.`